matchms/filtering/metadata_processing/require_valid_annotation.py

```diff
@@ -1,13 +1,63 @@
 import logging
 from matchms import Spectrum
 from matchms.filtering.filter_utils.smile_inchi_inchikey_conversions import (
-    convert_inchi_to_inchikey, convert_smiles_to_inchi, is_valid_inchi,
-    is_valid_inchikey, is_valid_smiles)
+    _check_fully_annotated, convert_inchi_to_inchikey, convert_inchi_to_smiles,
+    convert_smiles_to_inchi, is_valid_inchi, is_valid_inchikey,
+    is_valid_smiles)
+from matchms.filtering.metadata_processing.require_parent_mass_match_smiles import \
+    _check_smiles_and_parent_mass_match
 
 
 logger = logging.getLogger("matchms")
 
 
+def repair_not_matching_annotation(spectrum_in: Spectrum):
+    """When smiles and inchi do not match the annotation that matches parent mass will be kept"""
```

Comment:
This needs a more extensive docstring, since the name of the function "repair ... annotation" promises quite a lot ;)